npm_and_yarn/spec/dependabot/npm_and_yarn/update_checker/version_resolver_spec.rb

```diff
@@ -1319,48 +1319,49 @@
       let(:latest_allowable_version) { Gem::Version.new("16.3.1") }
 
       it "gets the right list of dependencies to update" do
-        expect(resolver.dependency_updates_from_full_unlock).
```

Comment:
Spotted this flake here https://github.com/dependabot/dependabot-core/runs/2030784173